### ISSUE TITLE
DD Tech: Send only a note to a contact, no time entry

### ISSUE THREAD

**Issue by Lowell-Syncrety** [DeskDirector/UserVoice#382]
When just going to Add Note in a ticket in DD Tech I don't see the "Send to Contact" and/or resources option. Could you add these? We would like to send a note without having to make it a time entry. There are many cases where this is useful:
- Quick question for the customer (could you do a reboot and let me know if that resolves?)
- Quick comment to let them know you'll be getting to the ticket soon.
- Streamline IT person wants to let his customer know he's escalating the ticket to us.

**Comment by Lowell-Syncrety** [comment]
I keep wanting to use DD Tech because it's so extremely quick to find tickets compared to CWM. But many times I'm just responding to a question about an update, or something I just need to send a note on. This happens more for sales tickets than service tickets, but even so... Had a customer ask about progress on a ticket, and the tech heading up the effort isn't in today. Just wanted to send them a quick note that they'll get an update tomorrow, but have to go find the ticket in CWM to do this very simple task.

**Comment by Andrew-Lahikainen** [comment]
@Lowell-Syncrety if you add a note using tech, do the customers not receive an email?

**Comment by Andrew-Lahikainen** [comment]
@Lowell-Syncrety just to let you know - there is no api that will allow these flags to be set. Also I don't think ConnectWise supports this behaviour when adding a note in their interface either:

![image](https://user-images.githubusercontent.com/10571313/73693891-e9d64e00-473b-11ea-930a-9018ba7b0ef7.png)

**Comment by Lowell-Syncrety** [comment]
> @Lowell-Syncrety if you add a note using tech, do the customers not receive an email?

No, and there's no selection for whom to send the note to. That selection (i.e. Contact) only shows up when using Time Entry.

**Comment by Lowell-Syncrety** [comment]
> @Lowell-Syncrety just to let you know - there is no api that will allow these flags to be set. Also I don't think ConnectWise supports this behaviour when adding a note in their interface either:
> 
> ![image](https://user-images.githubusercontent.com/10571313/73693891-e9d64e00-473b-11ea-930a-9018ba7b0ef7.png)

We send notes like this every day. CWM has the capability. Go to Discussion, click New Note. There you can choose to email to the Contact, CCs, Resources, or add manual CCs, and add attachments. These options are visible in the "Send Notes as Email" pod once you're in the window.

**Comment by Andrew-Lahikainen** [comment]
@Lowell-Syncrety just had a look at what ConnectWise is doing here. They are using an internal API which we cannot use without risk of breaking changes as generally these apis are closely tied with their interface. When you uncheck the Enter Time Record option, the internal API will transform the time entry model into a note and also processes emails as if it were creating a time entry.

The flag which controls this behavior is this:

![image](https://user-images.githubusercontent.com/10571313/73785086-801d7900-47fb-11ea-8144-a5a5ecfd9f66.png)

The V3 ConnectWise apis have no such feature for using the time entry api for adding notes. They do have a flag called processNotifications on the add note api which we do set for public notes, so your customers should be receiving an email. If not, then create a support ticket so we can investigate. I'm not 100% sure how this flag affects CCs though.

**Comment by Lowell-Syncrety** [comment]
OK, I'll send a support ticket, and comment back here once that's resolved, see if that works.
